LibraBFT/Impl/Consensus/EpochManager.agda

```diff
@@ -485,11 +510,11 @@ obmStartLoop self initializationOutput
         --(rm'' , to'') ← DAR.runOutputHandler rm' to pe o  oh
         rm'' ← pure rm -- TEMPORARY for previous two lines
         loop (setProcessor em rm'') {-to''-} rlec
-      (PMNewEpochManager em' newEpochInitializationOutput) → do
+      (PMNewEpoch em' newEpochInitializationOutput) → do
        eitherSD (em' ^∙ emObmRoundManager) ee $ λ rm → do
         -- (rm', to') ← DAR.runOutputHandler   rm  to pe newEpochInitializationOutput oh
         rm' ← pure rm -- TEMPORARY for previous line
-        loop (setProcessor em' rm') {-to'-} rlec -- TODO Set₁ != Set
+        loop (setProcessor em' rm') {-to'-} RSNothing -- IMPORTANT*** : reset RLEC state
```

Comment:
Yes, it is important, but then it's important to get _many_ things right :smile:.  So, I would keep the comment but remove "IMPORTANT***".  Great catch, btw!

Reply:
DONE